e2e-tests/pitr/run

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+test_dir=$(realpath $(dirname $0))
+. ${test_dir}/../functions
+
+
+run_recovery_check_pitr() {
+    local cluster=$1
+    local restore=$2
+    local backup=$3
+    local compare=$4
+
+    desc 'recover backup' $restore
+    kubectl_bin apply -f "$test_dir/conf/${restore}.yaml"
+    wait_backup_restore ${backup}
+    kubectl patch pxc pitr --type=merge -p "{\"spec\":{\"backup\":{\"pitr\":{\"enabled\":false}}}}"
+    kubectl_bin logs job/restore-job-${backup}-${cluster}
+    kubectl_bin delete -f "$test_dir/conf/${restore}.yaml"
+    wait_for_running "$cluster-proxysql" 1
+    wait_for_running "$cluster-pxc" 3
+    kubectl patch pxc pitr --type=merge -p "{\"spec\":{\"backup\":{\"pitr\":{\"enabled\":false}}}}"
```

Comment:
why do we need to call it 3 times?

Reply:
fixed https://github.com/percona/percona-xtradb-cluster-operator/pull/697/commits/867e25afaa236becf1253b02c8bf67e3c397219e